assemblies/platform-plugin/src/main/resources/resources/web/common-ui-require-js-cfg.js

```diff
@@ -38,6 +38,7 @@
   // TODO: This fallback logic is temporary, and can be removed when the remaining
   //    parts of the system rename the "service" plugin id to "pentaho/service".
   var requireTypes = requireCfg.config["pentaho/service"] || (requireCfg.config["pentaho/service"] = {});
+  var requireTypeInfo = requireCfg.config["pentaho/typeInfo"] || (requireCfg.config["pentaho/typeInfo"] = {});
```

Comment:
This two lines, specifically:
```js
requireTypeInfo["pentaho/visual/action/select"] = {alias: "select"};
requireTypeInfo["pentaho/visual/action/execute"] = {alias: "execute"};
```
 allow attaching to action events using short ids. Check how `EventSource` resolves given eventType names by passing them through the `pentaho/typeInfo` service.

The other `pentaho/type` declarations in this file, I just added them for completeness.